explainability-connectors/src/main/java/org/kie/trustyai/connectors/kserve/v2/TensorConverter.java

```diff
@@ -413,7 +409,7 @@ public static List<PredictionOutput> parseKserveModelInferResponse(ModelInferRes
                                     tensors.get(tensorIDX),
                                     names,
                                     perTensorSecondShape.get(tensorIDX),
-                                    raw, tensorIDX).get(0).getOutputs();
+                                    raw).get(0).getOutputs();
```

Comment:
This is a merge error, please rectify/don't change the tensor converter

Reply:
sorry, no idea what happened.  I reverted the PR copy to the main branch for this file.